README.md

```diff
@@ -205,14 +205,16 @@ It is also available on [YouTube](https://www.youtube.com/watch?v=NAZ2VeiN1N8)
 ## Running the dashboard locally
 
 Start the API first (port 8000):
-`cd backend`
+`cd api`
 `python main.py`
 
 Start the frontend (port 5137):
-`cd frontend`
+`cd dashboard/dashboard_1`
```

Comment:
should it be `dashboards`, if there are multiple?

Reply:
> should it be `dashboards`, if there are multiple?

changed it in the latest commit